PackagingCon2021-fpm/main.tex

```diff
@@ -0,0 +1,320 @@
+Brad~Richardson,
+John~Urban}
+\institute{\includegraphics[width=.5in]{Figures/logo-fortran}}
+\date{10th November, 2021\\PackagingCon}
+
+\begin{document}
+
+\setbeamercolor{normal text}{fg=light,bg=dark}
+\begin{frame}[plain]
+   \titlepage
+\end{frame}
+\setbeamercolor{normal text}{fg=dark,bg=light!50}
+
+\section{Motivation}
+
+\begin{frame}{Outline}
+
+   \vfill
+   \tableofcontents
+
+   \vfill
+\end{frame}
+
+\begin{frame}{What is Fortran?}
+   \begin{itemize}
+      \item Fortran was developed by John Backus' team at IBM in 1954--1957
+      \item created to ease translation of \alert{mathematical formulas} into machine code
+      \item the first \alert{high-level}, \alert{optimized}, and \alert{cross-platform} programming language
+      \item highly successful in scientific computing and engineering
+      \item many scientific applications and libraries are developed in Fortran
+      \item language still under active development, latest standard is \alert{2018}
+   \end{itemize}
+\end{frame}
+
+\begin{frame}{Motivation}
+   \begin{itemize}
+      \item building and distributing Fortran software is difficult
+         \begin{itemize}
+            \item[--] manual makefiles are seldom cross platform compatible
+            \item[--] autoconf is not working on native Windows
+            \item[--] learning curve for CMake is incredibly steep
+            \item[--] meson does not install module files automatically
+         \end{itemize}
+      \item writing of build files and adjustment for Fortran is time-intensive
+      \item difficult and tedious to reuse Fortran source code
+      \item system-wide installation dangerous for multiple Fortran compilers
+         (no ABI compatibility)
+      \item impossible to depend on a project without redistributing it
+         \\[2ex]
+      \item[\alert{▶}] create a \alert{Fortran-specific} build system and package manager to fix this
+      \item[\alert{▶}] make it easy to create \alert{reusable} Fortran libraries
+      \item[\alert{▶}] Fortran package manager (\alert{fpm})
+   \end{itemize}
+\end{frame}
+
+\begin{frame}{Adoption of fpm}
+   \begin{columns}
+      \begin{column}{.5\textwidth}
+         \includegraphics[width=\textwidth]{Figures/fpm-projects.pdf}
+         \vskip-2ex
+
+         \scriptsize
+         Data collected from GitHub and GitLab on 15th Sep, 2021  % TODO: Update
+      \end{column}
+      \hskip-1em
+      \begin{column}{.55\textwidth}
+         \begin{itemize}
+            \item today \alert{173} open source projects are using fpm  % TODO: Update
+            \item ca. \alert{160} unique developers have contributed\\ to those projects  % TODO: Update
+            \item \alert{62} contributors at the fpm project
+            \item code contributions from \alert{21} developers to fpm
+            \item hosted on GitHub (MIT licensed):\\\alert{https://github.com/fortran-lang/fpm}
+         \end{itemize}
+      \end{column}
+   \end{columns}
+\end{frame}
```

Comment:
I think this slide came too soon. I suggest moving it to after current slide 11 (First class dependencies)

Reply:
I'll move it to the summary section